admin_guide/managing_projects.adoc

```diff
@@ -376,3 +379,51 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+endif::openshift-enterprise,openshift-origin,openshift-dedicated[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-linmits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+If you want to enforce different limits on the number of projects that your
+users can create, apply the appropriate label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+
+.Example Project Limit for Users
+
+----
+'openshift.io/project-limit':3
+'openshift.io/project-limit':20
+----
+
```

Comment:
There are logs of labels can be added to one user. I think it is better list all the labels which can be added to some users.I check the master-config on ded environment, bellow selectors exist so far:
openshift.io/project-limit: '0'
openshift.io/project-limit: '1'
openshift.io/project-limit: '3'
openshift.io/project-limit: '5'
openshift.io/project-limit: '10'
openshift.io/project-limit: '20'
openshift.io/project-limit: '50'
openshift.io/project-limit: '100'
openshift.io/project-limit: 'unlimited'